quarkus/defaults/src/main/resources/application.properties

```diff
@@ -159,3 +159,7 @@ quarkus.index-dependency.guava.group-id=com.google.guava
 quarkus.index-dependency.guava.artifact-id=guava
 quarkus.index-dependency.protobuf.group-id=com.google.protobuf
 quarkus.index-dependency.protobuf.artifact-id=protobuf-java
+
+# force the locale, just in case the system's using another default locale
+quarkus.default-locale=en_US
```

Comment:
Could you mention in the PR description of subject line that we set default locale in Quarkus too?